frontend/src/styles/App.css

```diff
@@ -107,3 +107,21 @@ footer {
 .card-body {
   background-color: var(--light-color) !important;
 }
+
+.transactionContainer{
+  background-color: var(--light-color) !important;
+  margin-top: 8px !important; 
+}
```

Comment:
Let's rather reuse the above card-body style for this - check my suggested change in TransactionItem.js row 7.